client/src/components/LinkPicker/tests/LinkPickerMenu-test.js

```diff
@@ -0,0 +1,73 @@
+/* global jest, test */
+
+import React from 'react';
+import { render, fireEvent, act } from '@testing-library/react';
```

Comment:
```suggestion
import { render, fireEvent, screen } from '@testing-library/react';
```
Assuming that act() can be replace with screen.findByText()

Reply:
Updated